### ISSUE TITLE
Prǽstet

### ISSUE THREAD

**Issue by bbloomf** [gregorio-project/hyphen-la#8]
Prǽstet ought to hyphenate as Prǽ-stet, the same way that Præstet hyphenates as Præ-stet.

However, it is currently hyphenating as Prǽs-tet

**Comment by fradec** [comment]
In principle, it is not necessary to write the acute on a word of one or two syllables in Latin, so we did not write it in the files of the repo… that explain the difference.

**Comment by fradec** [comment]
After some tests, it appears that the problem comes from the accented character ǽ. We can circumscribe the verb præsto to 144 forms. Of these 144 forms, 14 are accented on the æ, 12 of which do not require to mark it since they contain only 2 syllables:

- præsta
- præstans
- præstant
- præstas
- præstat
- præstem
- præstent
- præster
- præstes
- præstet
- præsto
- præstor

and 2 forms that have 3 syllables and therefore need to mark the accent:

- prǽstiti
- prǽstitit

The verb forms that do not need accent are correctly hyphenated.

**Comment by fradec** [comment]
I will add all the verb forms in a test file and will take care of the correction of the patterns for those two forms, this should probably work also for accented two-syllable forms, for those who use them.

**Comment by fradec** [comment]
I searched all Latin words starting with *præst*, it gives about fifty words if we do not count the declensions or conjugations.
All are composed of the prefix *præ*, which gives a hyphenation in præ- without difficulty.
A slight doubt could subsist however for *præstino*. Goelzer gives an etymology from *præs*, but it is not very clear, and a badly dropped line break prevents to be sure of the hyphen (!). On the other hand, I found a book (*Les mots latins groupés d'après le sens et l'étymologie*, Bréal & Bailly, page 366) which gives as *stano* origin with the prefix *præ*.
If we take this for valid (according to the law of the "who says better?"), there is no exception to the hyphenation *præ-*.
